lib/softaculous_installer.php

```diff
@@ -27,107 +27,164 @@ public function __construct($logger)
     abstract public function install(stdClass $service, stdClass $meta);
 
     /**
-     * Gets a list of scripts available in softaculous
+     * Send an HTTP request.
      *
-     * @global type $SoftaculousScripts
-     * @global type $add_SoftaculousScripts
-     * @return array A list of scripts available in softaculous
+     * @param array $post The parameters to include in the request
+     * @param string $url Specifies the url to invoke
+     * @param string $method Http request method (GET, DELETE, POST)
+     * @return string An json formatted string containing the response
      */
-    protected function softaculousScripts()
+    protected function post(array $post, $url, $method = 'GET')
```

Comment:
Seems like a more apt name would be "request" or "makeRequest" rather than "post" since that is an HTTP verb that contradicts the default "GET" method

Reply:
You could also just have a method for each HTTP verb, i.e. "get", "post", "delete", etc.

Reply:
Updated to makeRequest